datafusion/wasmtest/src/lib.rs

```diff
@@ -17,9 +17,10 @@
 extern crate wasm_bindgen;
 
 use datafusion_common::{DFSchema, ScalarValue};
+use datafusion_expr::execution_props::ExecutionProps;
```

Comment:
Would it be possible to leave backwards compatibility `pub use` statements to avoid a breaking API change?

Specifically I am thinking of users  who had

```
use datafusion_optimizer::simplify_expressions::{ExprSimplifier, SimplifyContext};
```

If we put this in `datafusion_optimizer::simplify_expressions` I think existing code wouldn't need a change

```rust
// backwards compatibility
pub use datafusion_expr::simplify::SimplifyContext;
pub use datafusion_optimizer::simplify_expressions::ExprSimplifier;
```

Also  in `datafusion_physical_expr` something like
```rust
// backards compatbolituy
pub mod execution_props {
  pub use datafusion_expr::execution_props::ExecutionProps;
}
```

Reply:
Here is an example: https://github.com/apache/arrow-datafusion/blob/a1ae15826245097e7c12d4f0ed3425b25af6c431/datafusion/core/src/datasource/mod.rs#L38-L39